.bazelrc

```diff
@@ -33,6 +33,16 @@
 build --test_env=LC_CTYPE=en_US.UTF-8
 build --test_env=MPLBACKEND=Template
 
+# Location of the Gurobi license key file, typically named "gurobi.lic".
+# Setting this --test_env for all configurations is deliberate to improve
+# remote caching performance.
+test --test_env=GRB_LICENSE_FILE
```

Comment:
This does not make sense for the binary workflow.

Reply:
added `build --action_env=GRB_LICENSE_FILE` also.  i assume that was your intention?